editor/editor_inspector.h

```diff
@@ -361,7 +361,9 @@ class EditorInspectorArray : public EditorInspectorSection {
 		PanelContainer *panel = nullptr;
 		MarginContainer *margin = nullptr;
 		HBoxContainer *hbox = nullptr;
+		Button* move_up = nullptr;
 		TextureRect *move_texture_rect = nullptr;
+		Button* move_downp = nullptr;
```

Comment:
```suggestion
		Button *move_down = nullptr;
```
And rename where used